hyperjumptech/monika
# chore: add license to every ts and js file

### Description

- added license notice to every .ts and .js file
- added pre-commit script to add license notice to every .ts or .js file if it's not there yet


### How to test?

- create a new .ts file
- git commit
- a license notice will be added to the beginning of file
